models/maat/DESCRIPTION

```diff
@@ -0,0 +1,21 @@
+Package: PEcAn.MAAT
+Type: Package
+Title: PEcAn package for integration of the MAAT model
+Version: 1.4.5
+Date: 2016-03-30
+Author: Shawn Serbin, Anthony Walker
+Maintainer: Shawn Serbin <sserbin@bnl.gov>
+Description: This module provides functions to link the MAAT to PEcAn.
+Depends:
+    PEcAn.utils,
+    proto,
```

Comment:
@mdietze Correct!  MAAT needs proto but the PEcAn.MAAT package does not.  Therefore, we should be able to eliminate proto as a PEcAn dependency.  That is my mistake.  I will correct and send over a new pull request later.